island/settings.py

```diff
@@ -31,9 +31,8 @@
 
 ALLOWED_HOSTS = ['localhost', 'si.bitcrafter.net']
 
-EXTRA_ALLOWED_HOSTS = os.environ['EXTRA_ALLOWED_HOSTS']
-if EXTRA_ALLOWED_HOSTS:
-    ALLOWED_HOSTS = ALLOWED_HOSTS + [EXTRA_ALLOWED_HOSTS]
+if 'EXTRA_ALLOWED_HOSTS' in os.environ:
+    ALLOWED_HOSTS = ALLOWED_HOSTS + [os.environ['EXTRA_ALLOWED_HOSTS']]
```

Comment:
beyond the scope of this PR, but I'd point out that although the environment variable is plural, it only supports a single element. There's nothing here that would parse a particular input format and turn it into a Python list (for example, parsing a string containing 2 or more elements separated by commas). I will not be fixing this since that's out of scope